diff --git a/scripts/fuzzback.sh b/scripts/fuzzback.sh
index be36286..63bb44e 100755
--- a/scripts/fuzzback.sh
+++ b/scripts/fuzzback.sh
@@ -37,7 +37,6 @@ finder_split_cmd() {
     --no-multi \
     --print-query \
     --with-nth="3.." \
-    --color="$4" \
     --preview="$CURRENT_DIR/preview.sh $CAPTURE_FILENAME {}" \
     --preview-window="$preview_window"
 }
@@ -57,7 +56,6 @@ fzf_popup_cmd() {
     --no-multi \
     --print-query \
     --with-nth="3.." \
-    --color="$4" \
     --preview="$CURRENT_DIR/preview.sh $CAPTURE_FILENAME {}" \
     --preview-window="$preview_window"
 }